docs/development/writingzeppelininterpreter.md

```diff
@@ -32,17 +42,27 @@ Once you have build your interpreter, you can place your interpreter under direc
 
 ### Configure your interpreter
 
-You can configure zeppelin.interpreters property in conf/zeppelin-site.xml
-Property value is comma separated [INTERPRETER_CLASS_NAME]
+To configure your interpreter you need to follow these steps:
 
-for example, 
+1. create conf/zeppelin-site.xml by copying conf/zeppelin-site.xml.template to conf/zeppelin-site.xml 
 
-```
+2. Add your interpreter class name to the zeppelin.interpreters property in conf/zeppelin-site.xml
+
+  Property value is comma separated [INTERPRETER_CLASS_NAME]
+for example,
+  
+  ```
 <property>
   <name>zeppelin.interpreters</name>
   <value>org.apache.zeppelin.spark.SparkInterpreter,org.apache.zeppelin.spark.PySparkInterpreter,org.apache.zeppelin.spark.SparkSqlInterpreter,org.apache.zeppelin.spark.DepInterpreter,org.apache.zeppelin.markdown.Markdown,org.apache.zeppelin.shell.ShellInterpreter,org.apache.zeppelin.hive.HiveInterpreter,com.me.MyNewInterpreter</value>
 </property>
 ```
+3. start zeppelin by running ```./bin/zeppelin-deamon start```
+
+4. in the interpreter page, click the +Create button and configure your interpreter properties.
+Now you are done and ready to use your interpreter.
+
+Note that the interpreters shipped with zeppelin have a slightly different configuration process as they are registered by default without the use of the zeppelin-site.xml.
```

Comment:
Thanks for the corrections,  i will update the file tomorrow.
Why wont we ship with zeppelin-site.xml by default instead of .template?
This will be more strait forward with other configuration?
Eran

בתאריך יום ב׳, 13 ביולי 2015, 19:53 מאת Lee moon soo <
notifications@github.com>:

> In docs/development/writingzeppelininterpreter.md
> <https://github.com/apache/incubator-zeppelin/pull/155#discussion_r34484166>
> :
>
> >  <property>
> >    <name>zeppelin.interpreters</name>
> >    <value>org.apache.zeppelin.spark.SparkInterpreter,org.apache.zeppelin.spark.PySparkInterpreter,org.apache.zeppelin.spark.SparkSqlInterpreter,org.apache.zeppelin.spark.DepInterpreter,org.apache.zeppelin.markdown.Markdown,org.apache.zeppelin.shell.ShellInterpreter,org.apache.zeppelin.hive.HiveInterpreter,com.me.MyNewInterpreter</value>
> >  </property>
> >  ```
> > +3. start zeppelin by running ```./bin/zeppelin-deamon start```
> > +
> > +4. in the interpreter page, click the +Create button and configure your interpreter properties.
> > +Now you are done and ready to use your interpreter.
> > +
> > +Note that the interpreters shipped with zeppelin have a slightly different configuration process as they are registered by default without the use of the zeppelin-site.xml.
>
> More precisely, they are following the same configuration process. Only
> difference is default configuration
> <https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java#L397>
> includes interpreters shipped with Zeppelin, it is used when
> zeppelin-site.xml is missing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/155/files#r34484166>.
>